sql/schemas/bookbrainz.sql

```diff
@@ -46,6 +50,8 @@ ALTER TABLE bookbrainz.editor ADD FOREIGN KEY (gender_id) REFERENCES musicbrainz
 ALTER TABLE bookbrainz.editor ADD FOREIGN KEY (type_id) REFERENCES bookbrainz.editor_type (id);
 ALTER TABLE bookbrainz.editor ADD FOREIGN KEY (area_id) REFERENCES musicbrainz.area (id) DEFERRABLE;
 
+ALTER TABLE bookbrainz.editor ADD COLUMN privs INT NOT NULL DEFAULT 1;
```

Comment:
You can directly add a new column at the table declaration. No need to alter the table.

https://github.com/metabrainz/bookbrainz-site/pull/993/files#diff-78418986e4983535f6429f9d5766391c102e6c63830bca2071b8fad7051de628R32-R48

Reply:
I agree. Thanks!